packages/plugins/sentry/src/index.ts

```diff
@@ -77,11 +81,12 @@ export const useSentry = (options: SentryPluginOptions = {}): Plugin => {
   const includeRawResult = pick('includeRawResult', false);
   const includeExecuteVariables = pick('includeExecuteVariables', false);
   const renameTransaction = pick('renameTransaction', false);
-  const skip = pick('skip', () => false);
+  const skipOperation = pick('skipOperation', () => false);
+  const skipError = pick('skipError', () => false);
```

Comment:
As discussed the default value of `skipError` should be `(err: Error) => err instanceof EnvelopError` and exported from this plugin as `defaultSkipError`